Aula02/style_blog.css

```
@@ -0,0 +1,93 @@
+.blog-footer{
+    background-color: black;
+    padding: 10px;
+    font-size: 30px;
+    display: flex;
+    justify-content: space-between;
+}
+
+.menu {
+    background-color: black;
+    font-size: 30px;
+    margin: 0;
+    padding: 0;
+    display: flex;
+    justify-content: space-between;
+}
+  
+.menu li {
+    display: inline-block;
+    margin-right: 20px;
+}
+  
+.menu li a {
+    display: block;
+    padding: 10px;
+    text-decoration: none;
+}
+  
+.dropdown:hover .dropdown-menu {
+    display: block;
+}
+  
```

colocar uma width aqui pra fixar o tamanho do dropdown button
```suggestion
  .dropdown{
   position: relative;
   width: 200px;
  }
```